edb/lib/cal.edgeql

```diff
@@ -266,14 +318,51 @@ cal::date_get(dt: cal::local_date, el: std::str) -> std::float64
                 detail => (
                     '{"hint":"Supported units: century, day, ' ||
                     'decade, dow, doy, isodow, isoyear, ' ||
-                    'millenium, month, quarter, seconds, week, year."}'
+                    'millennium, month, quarter, seconds, week, year."}'
                 )
             )
         END
     $$;
 };
 
 
+CREATE FUNCTION
+cal::normalize_relative_hours(dur: cal::relative_duration)
```

Comment:
I don't like the name -- it goes against our naming convention where the typename is the first part of the name. Should be `duration_normalize_hours`.